_posts/2024-07-22-developer-update.md

```diff
@@ -0,0 +1,115 @@
+---
+author: Jody Garnett
+layout: post
+title: GeoServer 2024 Q3 Developer Update
+date: 2024-07-22
+categories:
+- Behind The Scenes
+---
+
+This is a follow up to [2024 roadmap]({% post_url 2024-01-03-roadmap %}) post outlining development opportunities.
+
+First of all thanks to developers and organisations that have responded with offers of in-kind contributions. We have not had enough capacity to organize in-kind contributions or fundraising and this blog post is an attempt to remedy.
```

Comment:
No apologies needed. Remember it's an open source project, what's already there is free, anything else, "contributions welcomed".

```suggestion
First of all thanks to developers and organisations that have responded with offers of in-kind contributions! This blog post is assessing the current situation and tries to pave a way forward to complete the Java 17/Jakarta EE/Spring 6 upgrades.
```

Reply:
yeah good advice, I take things too personally.